src/devices/KeyMatrix/README.md

```diff
@@ -0,0 +1,29 @@
+# Key Matrix
```

Comment:
you should also create another README.md in the samples folder and put some tiny snippets on how to use this